lib/classes/Swift/QueueSpool.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+use Interop\Queue\ExceptionInterface as PsrException;
+use Interop\Queue\PsrContext;
+use Interop\Queue\PsrQueue;
+
+class Swift_QueueSpool extends \Swift_ConfigurableSpool
+{
+    /**
+     * @var PsrContext
+     */
+    private $context;
+
+    /**
+     * @var PsrQueue
+     */
+    private $queue;
+
+    /**
+     * @param PsrContext      $context
+     * @param PsrQueue|string $queue
+     */
+    public function __construct(PsrContext $context, $queue = 'swiftmailer_spool')
+    {
+        $this->context = $context;
```

Comment:
This could be moved to the bottom of the block. It will make the code a (tiny) bit easier to read.